test/commands/test_parse.js

```diff
@@ -6,16 +6,44 @@
 const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
+const {execSync} = require('child_process');
 const {runSync, assertFilesExist, parserVersion, homeTag, weAreOnline} = require('../helpers');
 
 describe('parse', () => {
   before(weAreOnline);
 
+  const createTestProject = (dir) => {
+    fs.rmSync(dir, {recursive: true, force: true});
+    fs.mkdirSync(path.resolve(dir, 'src'), {recursive: true});
+    fs.writeFileSync(
+      path.resolve(dir, 'src/simple.eo'),
+      '# sample\n[] > simple\n'
+    );
+  };
+
```

Comment:
@vladislav-yevtushenko it's better not to leave empty lines inside function body: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html